make-options.json

```diff
@@ -140,6 +140,7 @@
         "UseDotNetV2",
         "UseNodeV1",
         "UsePythonVersionV0",
+        "UseRubyV1",
```

Comment:
We should probably add this to codeowners as well: https://github.com/Microsoft/azure-pipelines-tasks/blob/master/.github/CODEOWNERS

I don't think it will work with you since you're not a maintainer, you can probably add bryan and me though. Alternately you can request to become a maintainer of the vsts-dt-admin group, Bryan would approve that.

Reply:
Ill add myself as owner for now to catch any basic issues, then reassign to the proper ecosystem owner once they even are aware of its existence / are familiar with the changes

Reply:
That's fine - my only concern is will CodeOwners work even though you're not a maintainer? (Genuinely not sure)

Reply:
Im not sure. Are you aware of any automation that uses this information? I just assumed it was informational, and would serve as a way to route issues to me. It would be surprising if something tried to write to the repo on my behalf based on that info, since I could just put any name there

Does @stephenmichaelf know?

Reply:
It doesn't write to a repo, but it does make the codeowner a required reviewer. For example, in https://github.com/Microsoft/azure-pipelines-tasks/pull/9716, Ross was added as a reviewer automatically since he's a codeowner.

https://github.blog/2017-07-06-introducing-code-owners/

Reply:
I can't find any docs calling out that this only works for maintainers, I think we can probably assume it works for everyone. Worst case it doesn't and we just change it.